cmake/pheromones/common.cmake

```diff
@@ -0,0 +1,43 @@
+include(FetchContent)
+
+# Adding modules
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH})
+
+function(pheromones_fetch_populate)
+    if(PHEROMONES_PATH)
+        message(INFO "PHEROMONES_PATH specified, skipping fetch")
+    endif()
+
+    FetchContent_Declare(
+        pheromones
+        GIT_REPOSITORY https://github.com/SwarmUS/Pheromones
+        GIT_TAG        01bd792711588c425fe0d5e540cbdcff746f5fec
+        GIT_PROGRESS   TRUE
+    )
+    set(PHEROMONES pheromones)
+    string(TOLOWER ${PHEROMONES} PHEROMONES_L)
+
+    FetchContent_GetProperties(PHEROMONES POPULATED PHEROMONES_POPULATED)
```

Comment:
Change PHEROMONES to ${PHEROMONES} to properly resolve the POPULATED property. The same bug occurs is present in the freertos common.cmake. can you fix it at the same time?